ansible/inventory/inventory

```diff
@@ -0,0 +1,20 @@
+[microshift]
+microshift.example.com ansible_host= private_ip= # microshift node
```

Comment:
I got this error when filled in the ansible_host and private_ip:
```
fatal: [localhost]: FAILED! => {"msg": "[\"{{ hostvars['microshift'].ansible_host }}\", \"{{ hostvars['logging'].ansible_host }}\"]: \"hostvars['microshift']\" is undefined"}
```

```
microshift.example.com ansible_host=192.168.122.10 private_ip=192.168.122.10
```
How the host vars shall be populated?

Reply:
Sorry that was a bit hardcoded, my hosts did not have a hostname so it was looking for a host called `microshift`.
I've pushed a commit to dynamically build the host IPs based on the pattern specified in the inventory here.